src/js/components/Select/__tests__/Select-test.js

```diff
@@ -343,4 +343,17 @@ describe('Select', () => {
 
     expect(document.activeElement).toMatchSnapshot();
   });
+
+  test('open default state', () => {
+    render(
+      <Select
+        open
```

Comment:
I'm thinking we can use the `Custom Rounded` story in Select and change it to simply `Custom` and have the `open` prop set to true as an example. thoughts?